src/tests/ftest/harness/setup.py

```diff
@@ -22,4 +22,8 @@ def test_setup(self):
         :avocado: tags=hw,large
         :avocado: tags=harness,harness_setup_test,test_setup
         """
+        self.assertEquals(self.server_managers.storage_prepare_timeout, 60,
```

Comment:
(pylint-deprecated-method) Using deprecated method assertEquals()
(pylint-no-member) Instance of 'list' has no 'storage_prepare_timeout' member